internal/pkg/describe/webapp.go

```diff
@@ -16,12 +16,25 @@ import (
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 )
 
+const (
+	serviceLogicalID     = "Service"
+	targetGroupLogicalID = "TargetGroup"
+	taskDefLogicalID     = "TaskDefinition"
+)
+
 // WebAppURI represents the unique identifier to access a web application.
 type WebAppURI struct {
 	DNSName string // The environment's subdomain if the application is served on HTTPS. Otherwise, the public load balancer's DNS.
 	Path    string // Empty if the application is served on HTTPS. Otherwise, the pattern used to match the application.
 }
 
+// WebAppResources contains application resources created by cloudformation.
+type WebAppResources struct {
+	Service        string
+	TargetGroup    string
+	TaskDefinition string
+}
```

Comment:
Based on our discussion, maybe we can change this to be a generic struct for CFN resources
```go
type CfnResource struct {
    PhysicalID
    Type
}
```